.github/PULL_REQUEST_PROCESS.md

```diff
@@ -0,0 +1,41 @@
+# DNN Platform Pull Request Proceseses
+
+The goal of this document is to set standards for the review, processing, and approval of all Pull requests into the DNN_Platform GitHub repository, as well as associated commmunity open-source repositories.  These rules apply to ALL pull requests, regardless of the submitter, or complexity of the change.
+
+## Prerequisites
+The following items must all be true prior to a pull request being submitted to the DNN_Platform, if any of the following items are not true the pull request will be returned for edits referencing the specific missing item(s).
+
+* Change should represent an entire solution for the issue at hand.  Partial requests will NOT be processed.
+* Change should have a supporting issue logged on the DNN_Platform GitHub account, documenting the issue resolved, following the procedures outlined on the [Contribute Page](CONTRIBUTING.md)
+* Exception: Security items can be addressed individually, and should be initially communicated to the security@dnnsoftware.com email address for coordination
+* Pull request comment should contain at a minimum the following details (When creating, a default template will also prompt you for the proper information)
+--* Issue #, referenced such as “Fixes #24”
+--* Release Note: Suggested release note for the change such as “Improvement of user profile to improve performance when searching.”
+--* Testing Steps: Suggested testing steps for validation of the change
+* Pull request must pass the Continuous Integration (CI) Build.  Failure of the build will result in no futher review of the pull request until the build has been corrected.
+
+## Code Review Process
+Community review of submitted pull requests is encouraged, and all pull requests must be reviewed by at least **two (2)** community approvers before the change can be approved.  Once approved by two unique individuals the change can be merged by the secondary approval.  At the current time the following community members are designated approvers.
+
+* Mitchel Sellers ([mitchelsellers](https://github.com/mitchelsellers)) - Community Technology Advisory Group Lead
+* Oliver Hine ([ohine](https://github.com/ohine))
+* Brian Dukes ([bdukes](https://github.com/bdukes))
+* Peter Donker ([donker](https://github.com/donker)) - Community Developer Advisory Group Lead
+* Vicnec Masanas ([vmasanas](https://github.com/vmasanas))
+* Erik van Ballegoij ([erikvb](https://github.com/erikvb))
+
+An individual performing the code review should validate at a minimum the following.
```

Comment:
Do you want to mention that tests are greatly appreciated?

Reply:
How about:

 * Any tests related to the change have been updated
 * New tests covering the change are expected for areas of code already under test, and appreciated for areas of code not currently under test